iib/workers/tasks/build.py

```diff
@@ -117,6 +118,16 @@ def _build_image(dockerfile_dir: str, dockerfile_name: str, request_id: int, arc
         {'cwd': dockerfile_dir},
         exc_msg=f'Failed to build the container image on the arch {arch}',
     )
+    log.debug('Verifying that %s was built with expected arch %s', destination, arch)
+    buildah_inspect = json.loads(run_cmd(['buildah', 'inspect', destination]))
```

Comment:
Could we please reuse the `get_image_label` function here: https://github.com/release-engineering/iib/blob/master/iib/workers/tasks/utils.py#L1091

Reply:
skopeo inspect does not work as the image is not pushed yet

Reply:
I found a way to use, will update in next patch

Reply:
thank you!